### PR TITLE
Run pydocstyle on tests

### PR DESCRIPTION
**Pull request by nawatts**
Currently, PR checks run pydocstyle on files in the `gnomad` directory.

https://github.com/broadinstitute/gnomad_methods/blob/b5f83b81d42d95d3c1de3bd1b744b363690ac3ac/.github/workflows/ci.yml#L32-L33

Pre-commit hooks, however, run it on all Python files that changed in the commit.

https://github.com/broadinstitute/gnomad_methods/blob/b5f83b81d42d95d3c1de3bd1b744b363690ac3ac/.pre-commit-config.yaml#L15-L18

Thus, it's currently possible for a change to pass PR checks, get merged, and then cause pre-commit hooks to fail when it is pulled locally. This change brings PR checks and pre-commit hooks closer in line by running pydocstyle on tests in PR checks.

### CODE REVIEW

File: .pydocstylerc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [pydocstyle]
 convention = pep257
+match = .*\.py
```
Comment (ch-kr) [inline .pydocstylerc:3]: basic question, why is this line necessary if you've changed the workflow yaml to `pydocstyle gnomad tests`?

**Review by ch-kr — COMMENTED** [review]
small question

File: .pydocstylerc (hunk 1 above)
Comment (nawatts) [reply .pydocstylerc:3]: The default for `match` ignores files named `test_*.py`. So without this, `pydocstyle tests` doesn't actually check the test files contained in the `tests` directory.

http://www.pydocstyle.org/en/stable/usage.html#cli-usage

**Review by ch-kr — APPROVED** [review]
lgtm!
